src/components/Login.vue

```diff
@@ -5,41 +5,129 @@
+                    </div>
+                    <div class="row tasks">
+                        <div class="col-1 tasks"><img  src="../assets/images/Task4.svg" alt = "Starred comment" /></div>
+                        <div class="col-11"><p class= "sub">Recommends courses based on your needs</p> </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <img id= "hide" style="position:absolute" class="women" src="../assets/images/Person_planning.svg" alt = "women planning" />
+                </div>
+            </div>
+        </div>
 
-          <button @click="login" class="button">Log In</button> -->
+        <!-- DRAG -->
+        <div class="container-fluid plan p-0">
+            <div class="drag row no-gutters">
+                <div class = "col" >
+                    <img id= "hide" style="position:relative" class="preview" src="../assets/images/drag.svg" alt = "Dragging preview" />
+                </div>
+                <div class = "col comment" >
+                    <h1  class="head" style = "text-align: left;">Drag Your Course In</h1>
+                    <p class= "sub"> Courseplan’s intuitive interface recommends courses based on unfulfilled
+                        requirements and allows you to easily drag and drop them into your planner </p>
+                </div>
+            </div>
+        </div>
 
-          <!-- <div class="extras">
-          <a>Forgot Password</a>
-          <a>Create an Account</a>
-        </div> -->
-        </form>
-      </div>
-    </section>
+
+        <!-- SEMESTERS -->
+        <div class="container-fluid p-0 small">
+            <div class="row semester m-0">
+                <div class = "col-4 comment" >
+                    <h1 class= "head" style = "text-align: left;">Plan Your Semesters</h1>
+                    <p class= "sub">Use Courseplan’s semesterly planner to choose courses well in advance and ensure that you never miss a requirement</p>
+                </div>
+                <div class = "col" >
+                    <img id= "hide" class="schedule" src="../assets/images/schedule.svg" alt = "Plan preview" />
+                </div>
+            </div>
+        </div>
+
+        <!-- FIRST -->
+        <div class="container first m-0">
+          <div class = "container inside">
+              <div class="row justify-content-center">
+                <div class= "col justify-content-center">
+                  <h1 class= "head-center text-center">Be The First One To Use It</h1>
+                  <p class= "sub text-center" style= "padding: 30px; ">Gain early access by filling out your email below and help us grow into what you need!</p>
+                </div>
+              </div>
+              <div class="row email ">
+                  <div class="col-7  ">
+                      <input type="text" placeholder="Your Email Address" v-model="newUser.email" >
```

Comment:
Can we include email validation to ensure users only input email addresses? And maybe show a success/error message below the input/button if the email was successfully submitted or not?